packages/component/src/SendBox/SuggestedActions.js

```diff
@@ -67,7 +71,7 @@ const SuggestedActions = ({ className, suggestedActions = [] }) => {
 
   if (suggestedActionLayout === 'stacked') {
     return (
-      <div aria-label=" " aria-live="polite" role="status">
+      <div aria-label="test" aria-live="polite" role="status">
```

Comment:
Do we still want the aria-label to be test?

Reply:
oops
thanks